modules/graphql_content/src/TypeMatcher.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\graphql_content;
+
+class TypeMatcher {
+
+  /**
+   * Mapping of graphql types to drupal types.
+   *
+   * @var array
+   */
+  protected static $mapping = [
```

Comment:
I meant: Use a service parameter, not a whole service. Well actually a service WITH service parameters is even better ;). 

http://symfony.com/doc/current/service_container/parameters.html